iOSClient/Utility/NCContentPresenter.swift

```diff
@@ -60,63 +60,56 @@ class NCContentPresenter: NSObject {
         case info
     }
     
-    @objc private var lastErrorCode: Int = 0
-
     //MARK: - Message
     
-    @objc func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int, forced: Bool = false) {
+    @objc func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int) {
+        messageNotification(title, description: description, delay: delay, type: type, errorCode: errorCode, priority: .normal, dropEnqueuedEntries: false)
+    }
+    
+    func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int, priority: EKAttributes.Precedence.Priority = .normal, dropEnqueuedEntries: Bool = false) {
                        
-        // No notification message
-        if forced == false {
-            
-            if errorCode == -999 { return }         // Cancelled transfer
-            else if errorCode == 200 { return }     // Transfer stopped
-            else if errorCode == 207 { return }     // WebDAV multistatus
-            else if errorCode == 423 { return }     // WebDAV locked
-            else if errorCode == -1001 { return }   // Time out
-            else if errorCode == -1005 { return }   // Connection lost
-            else if errorCode == 0 && type == messageType.error { return }
-            
-            // No repeat message for:
-            if errorCode == lastErrorCode {
-                if errorCode ==  Int(CFNetworkErrors.cfurlErrorNotConnectedToInternet.rawValue) { return }
-            } else {
-                lastErrorCode = errorCode
-            }
-        }
+        // No notification message for:
+        if errorCode == -999 { return }         // Cancelled transfer
+        else if errorCode == 200 { return }     // Transfer stopped
+        else if errorCode == 207 { return }     // WebDAV multistatus
+        //else if errorCode == 423 { return }     // WebDAV locked
+        //else if errorCode == -1001 { return }   // Time out
+        //else if errorCode == -1005 { return }   // Connection lost
+        else if errorCode == NCGlobal.shared.errorNoError && type == messageType.error { return }
         
         DispatchQueue.main.async {
             switch errorCode {
             case Int(CFNetworkErrors.cfurlErrorNotConnectedToInternet.rawValue):
                 let image = UIImage(named: "networkInProgress")!.image(color: .white, size: 20)
-                self.noteTop(text: NSLocalizedString(title, comment: ""), image: image, color: .lightGray, delay: delay, name: "\(errorCode)")
+                self.noteTop(text: NSLocalizedString(title, comment: ""), image: image, color: .lightGray, delay: delay, priority: .max)
```

Comment:
Why is here `.max` hardcoded? Why not use the `priority` parameter? If I see it correctly, the parameter is not used elsewhere. If that's on purpose remove the `priority` parameter from the function.

Reply:
Because the message notification of NotConnectedToInternet required always the MAX priority

Reply:
Ok, in that case please remove the `priority` paramter from `messageNotification(...)`.

Reply:
Please ?

Reply:
The `priority` parameter is unused in [`messageNotification`](https://github.com/nextcloud/ios/blob/2e1bd0ac0165c4dce039d626d6cc6acc77964929/iOSClient/Utility/NCContentPresenter.swift#L69)

Reply:
no, it's used and it's important for some important notification and future notification

Reply:
It is not used in that function and causes confusion, because it does nothing. Please remove it.